### ISSUE TITLE
Respond to Invalid Session

### ISSUE THREAD

**Issue by ianagbip1oti** [princesslana/ErisCasper.java#64]
https://discordapp.com/developers/docs/topics/gateway#invalid-session

Respond appropriately.

In some instances, the appropriate response may be to disconnect (e.g., if the bot token is invalid).
In some instance, it may be to send an identify.

**Comment by CoreyShupe** [comment]
@ianagbip1oti does the codebase not respond to invalid sessions? Thought it did properly. Or did the merge just address the invalid token issue?

**Comment by ianagbip1oti** [comment]
I don't think we respond to all the situations described in the invalid session docs
